contract/test/test-agoric-basics-contract.js

```diff
@@ -0,0 +1,13 @@
+// @ts-check
+import '@endo/init';
+import { E } from '@endo/far';
+// eslint-disable-next-line import/no-unresolved -- https://github.com/avajs/ava/issues/2951
+import test from 'ava';
```

Comment:
I think we might need to do `import { test } from '@agoric/zoe/tools/prepare-test-env-ava.js';`...  but I am not up to date on latest best practices here and could use a 2nd opinion.